api/server/httputils/form.go

```diff
@@ -109,3 +114,24 @@ func ArchiveFormValues(r *http.Request, vars map[string]string) (ArchiveOptions,
 	}
 	return ArchiveOptions{name, path}, nil
 }
+
+// ParsePlatform parses the OCI platform from a form
+func ParsePlatform(platformJSON string) (*ocispec.Platform, error) {
```

Comment:
Oh; only thing I was just thinking; would it make sense to name this `DecodePlatform` or `UnmarshalPlatform`? It would possibly be a good counterpart to `EncodePlatform` or `MarshalPlatform` if we would ever decide to add that as a utility for the client side.

Reply:
Changed to `Decode` instead of `Unmarshal` because unmarshal functions conventionally take `[]byte` as an input.

Reply:
Good point as well 👍